flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -107,7 +108,7 @@
 	private static final AtomicReferenceFieldUpdater<Execution, ExecutionState> STATE_UPDATER =
 			AtomicReferenceFieldUpdater.newUpdater(Execution.class, ExecutionState.class, "state");
 	
-	private static final Logger LOG = ExecutionGraph.LOG;
+	private static final Logger LOG = LoggerFactory.getLogger(Execution.class);
```

Comment:
I think that is to some extend matter of taste how fine grained you split the loggers. Also, actually, if you create loggers by class or by job.

If you don't mind keeping it, I would like to keep it.